code/lib/cli/src/doctor/getMismatchingVersionsWarning.ts

```diff
@@ -30,9 +30,7 @@ export function getMismatchingVersionsWarnings(
         return Object.keys(frameworkPackages).includes(packageName);
       }
     );
-    const cliVersion =
-      getPrimaryVersion('@storybook/cli', installationMetadata) ||
```

Comment:
I don't have full context, but isn't Storybook 6 using the `@storybook/cli` and hence this change breaks behaviours if Storybook 6 user's are running the doctor command. I even don't know whether this is supported. @yannbf might have some context. I guess this is pretty niche, though.

Reply:
Would it be expected and supported for a user of storybook 6.x to be using `npx storybook@latest doctor`?